### PR TITLE
Issue #1006 Fix for oc-env used with the fish shell

### LINKED ISSUE #1006: oc-env command errors on fish shell
```fish
➜ ~ eval (minishift oc-env)
Unexpected end of string, quotes are not balanced
- (line 1): begin; set -gx PATH $PATH /home/q/.minishift/cache/oc/v1.5.1"; # Run this command to configure your shell:  # eval (minishift oc-env) 
                                                                             ^
from sourcing file -
	called on line 60 of file /usr/share/fish/functions/eval.fish

in function “eval”
	called on standard input

source: Error while reading file “-”

```
This appears to be due to an unnecessary closing quote on https://github.com/minishift/minishift/blob/master/pkg/util/shell/shell.go#L77. If that is indeed the solution
I shall submit a PR shortly.

### PR DESCRIPTION
**Pull request by quagliero**
Fix #1006  Remove quote from fish shell suffix

### CODE REVIEW

**Comment by budhrg** [comment]
@quagliero , could you updated your commit message as `Fix #1006 oc-env command errors on fish shell`? Thanks

**Comment by budhrg** [comment]
retest this please

**Comment by hferentschik** [comment]
OK to test

**Comment by quagliero** [comment]
Commit rebased and pushed with new message.

File: pkg/util/shell/shell.go [hunk 1]
```diff
@@ -74,7 +74,7 @@ func GetPrefixSuffixDelimiterForSet(userShell string, pathVar bool) (prefix, suf
 	switch userShell {
 	case "fish":
 		prefix = "set -gx "
-		suffix = "\";\n"
+		suffix = ";\n"
```
Comment (hferentschik) [inline pkg/util/shell/shell.go]: I am not a fish user, but I am wondering why you are choosing to remove the closing quote over  adding a potentially "missing" opening quote? Does fish have different rules around quoting?

Comment (quagliero) [inline pkg/util/shell/shell.go]: @hferentschik as far as I'm aware both would work, but the fish docs themselves don't use quotes (https://fishshell.com/docs/current/tutorial.html#tut_exports), and I never have when adding things to the $PATH in fish so didn't consider adding one :smile:

Comment (budhrg) [inline pkg/util/shell/shell.go]: @quagliero , please updated this section as mentioned here https://github.com/minishift/minishift/issues/1006#issuecomment-307301006

**Comment by LalatenduMohanty** [comment]
@budhrg we can close this as #1016 contains the commit. Not sure why we have not done it yet.

**Comment by hferentschik** [comment]
Let's wait with closing it.

**Comment by hferentschik** [comment]
Rebased and merged. Thanks @quagliero for reporting the issue and helping out with the pull request. Much appreciated.

**Pull request closed without merging**